backend/src/main/kotlin/org/loculus/backend/model/ReleasedDataModel.kt

```diff
@@ -42,6 +44,16 @@ class ReleasedDataModel(
             .map { computeAdditionalMetadataFields(it, latestVersions, latestRevocationVersions) }
     }
 
+    @Transactional(readOnly = true)
+    open fun getLastDatabaseWriteETag(): String {
+        val lastUpdateTime = UpdateTrackerTable.selectAll()
+            .mapNotNull { it[UpdateTrackerTable.lastTimeUpdatedDbColumn] }
+            .maxOrNull()
+            ?.replace(" ", "Z")
```

Comment:
Why is this necessary? Wouldn't it be better to use some sort of datetime object and format that to the desired format?

Reply:
No, we're using whatever is in the database - not converting from/to timestamp avoids the risks of timezone change etc.

We're leaving all logic in the database here, that's safest. It doesn't mater what it is. The reason I replace `" "` with `"Z"` is so that the bash script doesn't choke - otherwise no reason, it doesn't really matter what this etag is as long as it's ascending.

Reply:
The let's use numeric timestamps? They are guaranteed to be comparable.

Reply:
I think we can still convert to timestamp as we specify that we are using utc time in the sql tables and we can test that the tags are less than utc time now in our tests: 
```
lessThanOrEqualTo(Clock.System.now().toLocalDateTime(TimeZone.UTC).toTimestamp())
```
As Fabian says then we also know for certain that the value is numeric (the bash scripts require numeric values for the comparison)

Reply:
Don't we not have microsecond precision with this? There's really no reason to do any conversion here - every conversion adds a chance of introducing an error.

We treat whatever the db outputs as an etag - that's it. No need for _any_ processing.

Reply:
I still find the `Z` weird. Then let's simply remove whitespaces?

Reply:
Adding comment

Reply:
Comment added